default_configuration/templates/backend/config/secrets.yml.example

```diff
@@ -0,0 +1,57 @@
+# Be sure to restart your server when you modify this file.
+
+# Your secret key is used for verifying the integrity of signed cookies.
+# If you change this key, all old signed cookies will become invalid!
+
+# Make sure the secret is at least 30 characters and all random,
+# no regular words or you'll be exposed to dictionary attacks.
+# You can use `rake secret` to generate a secure secret key.
+
+# Make sure the secrets in this file are kept private
+# if you're sharing your code publicly.
+
+development:
+  secret_key_base: {{dev_secret_key_base}}
+  smtp:
+    address: smtp.{{app_name}}.com
```

Comment:
smtp server this will generally be something other than the appname.  e.g. sendgrid, google, or amazon simple smtp.   So I think this one is an inappropriate place to interpolate the appname.